### ISSUE TITLE
Limited RTE Editors

### ISSUE THREAD

**Issue by CharlesMx** [CharlesMx/mxCalendar#190]
It seems to me that the only RTE plugin I can use successfully with mxCalendar is TinyMCE, but that plugin uses TinyMCE 3.4.7. I have tried with other TinyMCE plugins like TinymceWrapper, but mxCalendar doesn't take to that either and does not display the TinyMCE interface. Even if I could use TinymceWrapper successfully, that is not an option with the company I work with, because TinymceWrapper only loads its TinyMCE resources from the cdn. Our company does not like cross-site loaded resources and there is no way for me to use a local source of TinyMCE unless I hack the plugin and don't ever have it check for updates because of the hack.

In short, I see and can use CkEditor with mxCalendars - is it possible that you could look into making sure that my event descriptions can be saved while using CkEditor, please?

**Comment by sebastian-marinescu** [comment]
I'm using it with Redactor and it works good.

**Comment by playcock** [comment]
Thank you Charles Mx!

**Comment by playcock** [comment]
Sebastian - Redactor looks interesting. Thank you for the share.

**Comment by sebastian-marinescu** [comment]
@playcock But I'm also using my fork, with many updates/fixes - I'm not sure if it works with this version.

**Comment by smahelos** [comment]
@CharlesMx  Hi, using ckEditor don't save description and content ..

**Comment by sebastian-marinescu** [comment]
Hi @smahelos, 
can you try with this version: https://github.com/sebastian-marinescu/mxCalendar/blob/master/_package/mxcalendars-1.1.12-rc2.transport.zip

**Comment by crimsonpixel** [comment]
I'm having same issue with RTE and using TinyMCE Rich Text Editor version 1.2.1

**Comment by crimsonpixel** [comment]
Any movement on this - I did try your download @sebastian-marinescu but was unsuccessful.  Think link may be broken.

**Comment by sebastian-marinescu** [comment]
Hi @crimsonpixel - the link doesn't seem broken. 
You have to click on "Download".

**Comment by crimsonpixel** [comment]
Hi @sebastian-marinescu I did download the zip file but it didn't update the existing mxCalendar extra. And yes I did search locally for packages having added to the core/packages/ folder.

**Comment by CharlesMx** [comment]
Let me test and I'll get an update in the next day or so, appreciate your
patience.

Cheers

**Comment by mor-victor** [comment]
> Let me test and I'll get an update in the next day or so, appreciate your patience. Cheers
> […](#)

Hi, any updates on this issue?
